pf-ui/src/main/webapp/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@adobe/css-tools": "^4.3.3",
```

Comment:
As @adobe/css-tools is not a direct dependency used by the UI it is better to locate this line under the "overrides" section of the package.json file

Reply:
Done!